addons/attributes/functions/fnc_buttonComponents.sqf

```diff
@@ -0,0 +1,39 @@
+#include "script_component.hpp"
+/*
+ * Author: mharis001
+ * Handles clicking the damage button.
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [] call zen_attributes_fnc_buttonComponents
+ *
+ * Public: No
+ */
+
+private _vehicle = GETMVAR(BIS_fnc_initCuratorAttributes_target,objNull);
+private _components = [];
+(getAllHitPointsDamage _vehicle) params ["_allHitPointNames", "_allHitSelectionNames", "_allHitPointsDamage"];
```

Comment:
Don't need the parentheses, and for variable names - I think: `["_hitPointNames", "_selectionNames", "_damageValues"]` might be better.

Reply:
But it's clearer with the parentheses. I'd agree with the variables though.

Reply:
I agree with Creep that the parentheses improve readability, so I have kept those.